### PR TITLE
SITE-1778: WP-CLI 2.11

### PR DESCRIPTION
**Pull request by pwtyler**
## Summary

**[Release Notes](https://docs.pantheon.io/release-notes)** - Adds release note for WP-CLI 2.11

https://pr-9209-documentation.appa.pantheon.site/release-notes/2024/09/wp-cli-2-11

## Blocked By
- [x] https://github.com/pantheon-systems/cos-framework-clis/pull/167

### CODE REVIEW

**Comment by pwtyler** [comment]
@jazzsequence & @scottbuscemi, I called out a couple of the features I thought were fun, but feel free to highlight any others from the linked release notes you deem noteworthy.

**Review by jazzsequence — COMMENTED** [review]
I would argue that the `--recently-active` filter, `--delete-unknown` for media resizes (particularly in light of https://docs.pantheon.io/release-notes/2024/08/prevent-filesystem-overload) and `wp user exists` are notable and worth calling out.

**Comment by pwtyler** [comment]
Added --delete-unknown and user exists but not looking to re-write all of Alain's blog post.

File: source/releasenotes/2024-09-06-wp-cli-2-11.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+This new command has been added to create multiple sites programmatically in a multisite installation. This is useful for testing or development purposes.
+
+* **GitHub Release Installation Support:**
+The extension command now supports installing plugins and themes directly from GitHub releases.
+
+* **Adds minor and patch limitations to theme updates:**
+Now, like plugins, themes can be updated only to their latest minor or patch releases
+
+* **New "user exists" command**
+A new `wp use exists` command allows for quick checks on whether a user exists in the WordPress database.
```
Comment (scottbuscemi) [inline source/releasenotes/2024-09-06-wp-cli-2-11.md:23]:
```suggestion
A new `wp user exists` command allows for quick checks on whether a user exists in the WordPress database.
```

**Review by scottbuscemi — COMMENTED** [review]
typo - otherwise LGTM

**Comment by IngridKwok** [comment]
@pwtyler Made some minor adjustments.

**Pull request merged**